lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -258,6 +289,26 @@ def self.available_options
                                  is_string: false,
                              default_value: false),
 
+          FastlaneCore::ConfigItem.new(key: :mapping,
+                                  env_name: "APPCENTER_DISTRIBUTE_ANDROID_MAPPING",
+                               description: "Path to your Android Proguard or R8 mapping.txt",
```

Comment:
I think we should not reference ProGuard or R8 here – if we do, please spell it "ProGuard" in public facing docs.

If we need something more specific than "mapping" we can call if "deobfuscation mapping"

Reply:
switched to simple version "Path to your Android mapping.txt"